### DISCUSSION TITLE
Releases UI folds notes by default, which is unsuitable for releasing downloads to end users with important warnings

### DISCUSSION THREAD

**Discussion by hai-vr** [community/community#6230] [Other Feature Feedback, Questions, & Ideas] [answered]
In our community of content creators around Unity and VRChat development, we often have to insert in our release notes warnings to end users who download the releases.

Our end users often come to us with support requests about our releases breaking stuff, because they don't follow prerequisites of installing the release, such as making sure a specific SDK is installed on their platform, or making sure they have backups because of a breaking change meaning migration to a major version is trickier than an regular upgrade.

Since we are dealing with game art assets, our end users are often players, artists, or users who are not necessarily tech savvy enough to make regular backups and use version control nor know how to use one. This can lead to loss of data, which no one wants.

We use formatting to provide content warnings, without it we get support requests from end users who did not follow instructions properly if they were less visible.

In my personal case, a large portion of my users are non-English speaking Japanese users, which requires me putting a warning in Japanese.

Therefore we try to put warnings in the header of the releases notes to protect our users, and ourselves from any backlash resulting from a mishap install.

This leads to notes like this on the live version of the releases UI, which looks atrocious but is necessary:

(note: I am not speaking on behalf of the people represented in those repositories)

![image](https://user-images.githubusercontent.com/60819407/136863060-fd8f5a8b-7e02-4de3-9174-15ac931aa47a.png)

![image](https://user-images.githubusercontent.com/60819407/136863089-ef23133e-b80e-41a9-b307-6fbf4ce011c9.png)

![image](https://user-images.githubusercontent.com/60819407/136863166-59bc4b5f-d8f2-43d1-a8bf-6a486e1d71dc.png)

----

With the new UI, these warnings are partially occluded:

![image](https://user-images.githubusercontent.com/60819407/136863490-3559ac70-f8e3-4266-b4b9-d3c88b5a5a95.png)

![image](https://user-images.githubusercontent.com/60819407/136863521-ec6949fb-8c6e-4c98-a87e-86bc8bb63538.png)

I am concerned that I may have to deal with support requests and possibly users losing data due to this UI change.

**Comment by MylesBorins** [comment, marked as answer]
We are making a couple of changes to the list view including making the latest release show the full notes, would this satisfy your use case?

**Reply by hai-vr** [reply]
Yes, I think it would ⭐

**Reply by MylesBorins** [reply]
As a quick update we've shipped a lot of changes to the list view, PTAL! At this time we do not expand the latest release by default, but that is mainly because there is other pending work to make the `latest` release more accurate. Once we figure out how to make that consistent / expected we plan to make the `latest` release expanded by default
